Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/AbstractSqlEamDb.java

```diff
@@ -1241,7 +1241,7 @@ public CorrelationAttribute getCorrelationAttribute(CorrelationAttribute.Type ty
                 correlationAttribute.addInstance(artifactInstance);
             }
         } catch (SQLException ex) {
-            throw new EamDbException("Error getting notable artifact instances.", ex); // NON-NLS
+            throw new EamDbException(Bundle.AbstractSqlEamDb_exceptionMessage_errorGettingNotableArtifactInstances(), ex); // NON-NLS
```

Comment:
Exceptions should not be bundle messages

Reply:
Removed